content/blog/argocd-canary.en.md

```diff
@@ -0,0 +1,499 @@
+---
+Title: Canary deployment with Argo
+Date: 2020-04-28
+Category: Kubernetes
+Summary: Introduction to ArgoCD and Argo-Rollout, Canary Deployment, rollback, everything *as code* !
+Author: Romain Guichard
+image: images/argo/argo-logo.png
+lang: en
+---
+
+Once again, we are going to take about GitOps... In [our last
+article](https://particule.io/blog/cncf-argo/), we briefly talked about Argo
+and [its adoption by the CNCF](https://www.cncf.io/projects/). As a reminder,
+Argo is a Continuous Delivery tools suite:
+
+- ArgoCD
+- Argo Workflow
+- Argo Rollout
+- Argo Event
+
+I would advise you to read or read again our article:
+mémoire :
+
+<https://particule.io/blog/cncf-argo/>
```

Comment:
maybe we should wait until this article is translated in english. I'll take care of it